osf/external/messages/celery_publishers.py

```diff
@@ -31,8 +31,8 @@ def publish_merged_user(user):
         _publish_user_status_change.s(
             body={
                 'action': 'merge',
-                'user_uri': user.get_semantic_iri(),
-                'merged_user_uri': user.merged_by.get_semantic_iri(),
+                'into_user_uri': user.get_semantic_iri(),
+                'from_user_uri': user.merged_by.get_semantic_iri(),
```

Comment:
If I'm reading correctly, this is backwards: https://github.com/CenterForOpenScience/osf.io/blob/c9774718fdaf028d1d123c27d6b82970d238904d/osf/models/user.py#L243